packages/block-directory/src/plugins/installed-blocks-pre-publish-panel/index.js

```diff
@@ -4,17 +4,14 @@
 import { _n, sprintf } from '@wordpress/i18n';
 import { useSelect } from '@wordpress/data';
 import { blockDefault } from '@wordpress/icons';
+import { PluginPrePublishPanel } from '@wordpress/editor';
 
 /**
  * Internal dependencies
  */
 import CompactList from '../../components/compact-list';
 import { store as blockDirectoryStore } from '../../store';
 
-// We shouldn't import the editor package directly
-// because it would include the wp-editor in all pages loading the block-directory script.
```

Comment:
What other pages is this referring to?

Reply:
The original comment referred to the `wp-edit-post` package, accidentally loading it on other pages like Site Editor.

The context got lost somewhere in a string replacing. Full details can be found here - #45895.